experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -50,7 +51,7 @@ export class TinyliciousClient {
     public async createContainer(
```

Comment:
oh, we'll need to make this change in FRS container as well before i can change this method name

Reply:
I really want to avoid the detached terminology. If we have a `createDetachedContainer` it implies there should a be a `createAttachedContainer` call. Instead creating always returns you something that needs to be attached. Calling it detached is just redundant if it always needs to be attached.